sql/core/src/main/java/org/apache/spark/sql/execution/vectorized/ColumnVector.java

```diff
@@ -807,6 +809,11 @@ public final int appendStruct(boolean isNull) {
   public final boolean isArray() { return resultArray != null; }
 
   /**
+   * Marks this column as being constant.
+   */
+  public final void setIsConstant() { isConstant = true; }
```

Comment:
Should we have a special ColumnVector for constants (that always return the same value )?